src/main/resources/com/google/api/codegen/java/method_sample.snip

```diff
@@ -53,22 +65,35 @@
 @end
 
 @snippet syncMethodSampleCode(apiMethod)
-  {@initCode(apiMethod.initCode)}
+  {@syncMethodSampleCode(apiMethod, apiMethod.initCode)}
+@end
+
+@snippet syncMethodSampleCode(apiMethod, initCode)
+  {@initCode(initCode)}
   @if apiMethod.hasReturnValue
     {@apiMethod.responseTypeName} response = {@sampleSyncMethodCall(apiMethod)};
   @else
     {@sampleSyncMethodCall(apiMethod)};
   @end
 @end
 
+# TODO(vchudnov-g): Deprecate this override
```

Comment:
Are all such overrides being deprecated? Also, deprecate or don't deprecate, don't "promise to deprecate."

Reply:
Yes. And fair enough.
Removed the now unused overloads.